action.yml

```
@@ -0,0 +1,16 @@
+name: 'Install dfx'
+description: 'Install the dfx utility at a particular version'
+inputs:
+  dfx-version:
+    description: >
+      The dfx version to install. This value will be used to populate
+      the DFX_VERSION variable that will be handled by the installation
+      script "install.sh".
+    default: '0.14.1'
+
+runs:
+  using: composite
+  steps:
+    - name: Install dfx
+      shell: sh
+      run: DFX_VERSION=${{ inputs.dfx-version }} sh -ci "$(curl -fsSL https://internetcomputer.org/install.sh)"
```

Could you please tweak it a little bit? I just realized that in the current form, we would have to bump the `default: ...` value whenever we create a new dfx release. Instead, let's set `default: ` to `'latest'` and do this instead
```suggestion
      run: |
        if [[ "${{ inputs.dfx-version }}" == "latest" ]]; then
          sh -c "$(curl -fsSL https://internetcomputer.org/install.sh)"
        else
          DFX_VERSION="${{ inputs.dfx-version }}" sh -c "$(curl -fsSL https://internetcomputer.org/install.sh)"
        fi
```

---

I kept `-ci` instead of `-c` as specified in the README.